Random fixes and tweaks for July update

hammmurabi opened the pull request:
just some random small tweaks to some trees
big things are:

focus reshuffle for portugal democratic path

hammmurabi committed:
random changes central

ilkober commented on common/national_focus/fake.txt:
Why is it there?

hammmurabi replied:
from my understanding if you remove a focus thats in vanilla u have to put it in fake.txt, is that not correct? i removed the "protect chinese civillians" focus because it made absolutely no sense and was just cluttering the tree

ilkober commented on common/national_focus/r56_manchukuo.txt:
One of them refers now to subject, the other one is free to take, why?

hammmurabi replied:
prevent player from being locked out of entire tree if gains independence, for example from japan going democratic/communist

that path is weaker than the other so i made it be the one you can do without being a puppet

ilkober replied:
No, there is no need to place it there
Just remove all the references to the focus if it comes up in our files and it usually should be fine

hammmurabi replied:
ok

ilkober replied:
For this (Manchukou), you could add a check for "having high relations (50/70+) with Japan" - that way a player can still access the tree if they gained independence in peaceful way

hammmurabi replied:
hmm ok can do

hammmurabi committed:
okok

hammmurabi committed:
some more stuffs

hammmurabi committed:
Update ethiopia.txt

hammmurabi committed:
Update infantry.txt

hammmurabi committed:
tank balancing

makes Turret Shield and fixed superstructure forbid the Tank role so they can only be used in AT and arty which makes much more sense

name of tank module file changed in order to override vanilla

hammmurabi committed:
trust

hammmurabi committed:
Update usa.txt

ilkober committed:
Merge branch 'Beta-Edits' into roblox-random-fixes
